config/salt/minion.d-ca/signing_policies.conf

```diff
@@ -3,8 +3,8 @@ x509_signing_policies:
     - minions: '*'
     - signing_private_key: /etc/pki/private/ca.key
     - signing_cert: /etc/pki/ca.crt
-    - basicConstraints: "CA:false"
-    - keyUsage: "nonRepudiation, digitalSignature, keyEncipherment"
+    - basicConstraints: "critical CA:false"
+    - keyUsage: "critical nonRepudiation, digitalSignature, keyEncipherment"
```

Comment:
This is just the syntax salt wants, I know it's weird, but it is :) Heres an example from their docs:

    x509_signing_policies:
      www:
        - minions: 'www*'
        - signing_private_key: /etc/pki/ca.key
        - signing_cert: /etc/pki/ca.crt
        - C: US
        - ST: Utah
        - L: Salt Lake City
        - basicConstraints: "critical CA:false"
        - keyUsage: "critical cRLSign, keyCertSign"
        - subjectKeyIdentifier: hash
        - authorityKeyIdentifier: keyid,issuer:always
        - days_valid: 90
        - copypath: /etc/pki/issued_certs/